pages/contact.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CONTACT</title>
+</head>
+<body>
+    <header>
+        <div>
+          <img src="../images/logo.png" alt="logo" width="400" height="auto" />
+        </div>
+    
+        <style>
+          body {
+            font-family: Arial, sans-serif;
+            margin: 0;
+          }
+    
+          nav {
+            background-color: #333;
+            overflow: hidden;
+          }
+    
+          nav a {
+            display: inline-block;
+            color: white;
+            text-align: center;
+            padding: 14px 16px;
+            text-decoration: none;
+          }
+    
+          nav a:hover {
+            background-color: #ddd;
+            color: black;
+          }
+        </style>
+        <nav>
+          <a href="../index.html">HOME</a>
+          <a href="../pages/about.html">ABOUT</a>
+          <a href="../pages/contact.html">CONTACT</a>
+          <a href="../faqs.html">FAQs</a>
+        </nav>
+      </header>
+        <section>
+          <h3>CONTACT US</h3>
+          <form
+  action="https://formspree.io/f/mbjvnrnl"
+  method="POST"
+>
```

Comment:
🌟 Great job implementing a functional Contact Me form, it is a valuable addition to user engagement